filebeat/docs/modules/apache.asciidoc

```diff
@@ -84,3 +84,14 @@ include::../include/var-paths.asciidoc[]
 For a description of each field in the module, see the
 <<exported-fields-apache,exported fields>> section.
 
+=== Virtual Host
+
+See customlog documentation  https://httpd.apache.org/docs/2.4/en/mod/mod_log_config.html
+Add %v config in httpd.conf in log section
```

Comment:
I would usually say that we only support default log formats in included pipelines, but I think that it is ok to also support this format as seems to be frequently used, and is included in the official documentation.